providers/src/airflow/providers/celery/executors/celery_executor.py

```diff
@@ -163,8 +162,12 @@ def __getattr__(name):
 
 CELERY_CLI_COMMAND_PATH = (
     "airflow.providers.celery.cli.celery_command"
-    if Version(airflow_version) >= Version("2.8.0")
-    else "airflow.cli.commands.local_commands.celery_command"
+    if AIRFLOW_V_2_8_PLUS
+    else (
+        "airflow.cli.commands.local_commands.celery_command"
+        if AIRFLOW_V_3_0_PLUS
```

Comment:
It might be worth switching this to flat if/elses. Much easier to determine what's happening vs nested ternary.

Case in point, won't "3.0" also be true for 2_8_PLUS, so we will never fall into this else for the 3.0 command?

Reply:
Adjusted to flat `if/elif/else`. It was nested so I left it that way while indeed putting it in the wrong place :sweat_smile: Thanks!